.nix/config.nix

```diff
@@ -0,0 +1,83 @@
+{
+  ## DO NOT CHANGE THIS
+  format = "1.0.0";
+  ## unless you made an automated or manual update
+  ## to another supported format.
+
+  ## The attribute to build, either from nixpkgs
+  ## of from the overlays located in `.nix/coq-overlays`
+  attribute = "coq";
+
+  ## If you want to select a different attribute
+  ## to serve as a basis for nix-shell edit this
+  shell-attribute = "coq-full";
+
+  ## Maybe the shortname of the library is different from
+  ## the name of the nixpkgs attribute, if so, set it here:
+  # pname = "{{shortname}}";
+
+  ## Lists the dependencies, phrased in terms of nix attributes.
+  ## No need to list Coq, it is already included.
+  ## These dependencies will systematically be added to the currently
+  ## known dependencies, if any more than Coq.
+  ## /!\ Remove this field as soon as the package is available on nixpkgs.
+  ## /!\ Manual overlays in `.nix/coq-overlays` should be preferred then.
+  # buildInputs = [ ];
+
+  ## Indicate the relative location of your _CoqProject
+  ## If not specified, it defaults to "_CoqProject"
+  # coqproject = "_CoqProject";
+
+  ## select an entry to build in the following `bundles` set
+  ## defaults to "default"
+  default-bundle = "default";
+
+  ## write one `bundles.name` attribute set per
+  ## alternative configuration, the can be used to
+  ## compute several ci jobs as well
+  bundles.default = {
+
+    coqPackages = {
+      bignums.override.version = "master";
+      CoLoR.override.version = "master";
+      CompCert.override.version = "master";
+      iris.override.version = "master";
+      mathcomp.override.version = "master";
+      stdpp.override.version = "master";
+      unicoq.override.version = "master";
+      VST.override.version = "master";
```

Comment:
This list will need to be expanded to encompass all the projects that we test in CI and for which we could need the dependencies to prepare an overlay. Some of them are not yet in nixpkgs and should be added there (or in `.nix/coq-overlays`).